### PR TITLE
Update files for Pandeia v1.6

### PR DESCRIPTION
**Pull request by ariedel**
This PR updates the various files to install and use Pandeia v1.6 and its associated Roman data.

There's an extra PYSYN_CDBS download this time, and as of when I write this, it's not the complete necessary file.

### CODE REVIEW

**Comment by robelgeda** [comment]
Hi @ariedel , is this the cycle 9 data?

**Comment by ariedel** [comment]
It is not. It's a new thermal model, WebbPSF 0.9 PSFs, and renaming wfirst to roman internally.

File: README.md [hunk 1]
```diff
@@ -135,20 +135,22 @@ Next, create a new directory somewhere with plenty of space to hold the referenc
 To obtain the [reference data](https://stsynphot.readthedocs.io/en/latest/#installation-and-setup) used for synthetic photometry, you will need to retrieve them via FTP. The `curl` command line tool can be used as follows to retrieve the archives:
 
 ```
-(roman_tools) $ curl -OL ftp://ftp.stsci.edu/cdbs/tarfiles/synphot1.tar.gz    # 85 MB
-(roman_tools) $ curl -OL ftp://ftp.stsci.edu/cdbs/tarfiles/synphot2.tar.gz    # 34 MB
-(roman_tools) $ curl -OL ftp://ftp.stsci.edu/cdbs/tarfiles/synphot5.tar.gz    # 505 MB
+(roman_tools) $ curl -OL https://archive.stsci.edu/hlsps/reference-atlases/hlsp_reference-atlases_hst_multi_everything_multi_v5_sed.tar    # 85 MB
+(roman_tools) $ curl -OL https://archive.stsci.edu/hlsps/reference-atlases/hlsp_reference-atlases_hst_multi_star-galaxy-models_multi_v3_synphot2.tar    # 34 MB
+(roman_tools) $ curl -OL https://archive.stsci.edu/hlsps/reference-atlases/hlsp_reference-atlases_hst_multi_pheonix-models_multi_v2_synphot5.tar    # 505 MB
+(roman_tools) $ curl -OL https://archive.stsci.edu/hlsps/reference-atlases/hlsp_reference-atlases_jwst_multi_etc-models_multi_v1_synphot7.tar
```
Comment (ariedel) [inline README.md]:
```suggestion
(roman_tools) $ curl -OL https://archive.stsci.edu/hlsps/reference-atlases/hlsp_reference-atlases_jwst_multi_etc-models_multi_v1_synphot7.tar # 9 MB
```

**Comment by ariedel** [comment]
Verified the synphot7 tarfile is now correct.

**Review by robelgeda — APPROVED** [review]
looks good to me, thank you! 
You can merge when ready.

**Pull request merged**